### DISCUSSION TITLE
Which soft was used to make graphs in README.MD ?

### DISCUSSION THREAD

**Discussion by kyshel** [juicedata/juicefs#58] [Q&A] [answered]
**Hi, guys. I like the graphs（show below） in readme.md file. It's clean, sharp and comfort. Can anyone tell me which soft may achieve this?**

![image1](https://github.com/juicedata/juicefs/raw/main/docs/images/juicefs-arch.png)
![image2](https://github.com/juicedata/juicefs/raw/main/docs/images/juicefs-storage-format.png?raw=true)

**Comment by xiaogaozi** [comment, marked as answer]
Google Drawings 😃
